data/testsuite

```diff
@@ -498,3 +498,229 @@ dalla bat kææwa
 dalla bat kææwa
 dog.NOM rice.NOM eat.PST
 `The dog ate rice`
+
+### Agreement in Coordinated NPs
+
+# 1 NP Coordination
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Coordination Agreement}
+Ranjitui Chitrai bat kan2wa
```

Comment:
The lab instructions say we are not supposed to use proper nouns in our examples so I would change these names to some common nouns we are already using